### ISSUE TITLE
About training AUC results

### ISSUE THREAD

**Issue by victorfica** [ninglab/mhcrank#2]
Hi! I was able to run the training step but i got training AUC = 0.5 in  fold_2 (reported in `training_results.csv`). Is this behavior correct? The mean test AUC i got was 0.66273. I used default hyperparameters and the following command (im running in my laptop): 
`python mhc_rank/train_processing_models_command.py --out-models-dir output/training --continue-incomplete`

Also the min loss im getting is in the range 0.61-0.68.
